packages/block-library/src/cover/style.scss

```diff
@@ -1,3 +1,13 @@
+:root {
+	.wp-block-cover-image,
+	.wp-block-cover {
+		padding-top: var(--wp--padding-top, 1em);
+		padding-right: var(--wp--padding-right, 1em);
+		padding-bottom: var(--wp--padding-bottom, 1em);
+		padding-left: var(--wp--padding-left, 1em);
```

Comment:
I'm a little bit concerned about the usage of CSS  variables here instead of inline styles.

If you include a cover image inside  a cover  image, the  padding will apply to both. While this might seem like an edge case,  it's not when it comes to Group block. We had the same issue when we tried CSS variables for colors and we switched  back to inline styles for this reason.

Reply:
Haii @youknowriad !!

Thank you for taking a look once again! I really appreciate it 🙏 

Adding the `padding` inline style directly caused it to render on the incorrect wrapping `div`, (see below)

<img width="683" alt="Screen Shot 2020-05-05 at 1 01 14 PM" src="https://user-images.githubusercontent.com/2322354/81093755-8b527480-8ed0-11ea-95aa-4e6d28899206.png">

(Padding `30px` is being applied to the first wrapper `div`, rather than the Cover `div`)

I believe you had suggested that the best solution was to refactor things using a `lightBlockWrapper`, which offers more flexibility in terms of where/how styles are applied.

Since that isn't the case (as of now), the only way I was able to get it working in a reliable way was to use CSS variables.

Reply:
> Since that isn't the case (as of now), the only way I was able to get it working in a reliable way was to use CSS variables.

It's  still on my  mind to refactor the cover to use the ligher block DOM. One blocker have been solved (the  align hook) but there's another funny  one :). The first element wrapper of the Cover block is  rendered by a third-party library  (resizeable-box). So the path forward is to propose a  PR for that library to support a custom `tagName` (I already did a PR on that repo before), or fork that library temporarily. I'll make a note to look at that soon (hopefully tomorrow). Please don't hesitate to ping me about it.